include/clap/ext/draft/location.h

```diff
@@ -0,0 +1,73 @@
+#pragma once
+
+#include "../../color.h"
+#include "../../plugin.h"
+#include "../../string-sizes.h"
+
+// This extension allows a host to tell the plugin more about its position
+// within a project or session.
+
+static CLAP_CONSTEXPR const char CLAP_EXT_LOCATION[] = "clap.location/1";
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+enum {
+   // Represents a document/project/session.
+   CLAP_PLUGIN_LOCATION_PROJECT = 1,
```

Comment:
Should this be required to be the first element in the array passed to `set_location`, if it is provided?

Reply:
I don't think it should, because if your host is just a device chain, it doesn't have a track or a project.